sources/packages/api/src-sql/DeleteCascades/Add-cascade-rules.sql

```diff
@@ -0,0 +1,203 @@
+-- Create reference between application_exception_requests and application_exceptions --
```

Comment:
When the rollback script looks like something that we are adding we usually add some prefixes like `Rollback-drop-cascade-rules.sql`.

Reply:
I will update